src/apps/experimental/routes/user/display/types.ts

```diff
@@ -0,0 +1,29 @@
+export interface JellyfinTheme {
```

Comment:
We should reuse the Theme interface in `webConfig`: https://github.com/jellyfin/jellyfin-web/blob/master/src/types/webConfig.ts

Reply:
Ah nice. I did some minor edits there to match and removed this in favor of that one.